.github/workflows/build_symengine.yml

```diff
@@ -146,6 +146,11 @@ jobs:
       run: pip install conan
     - name: create profile
       run: conan profile new tket --detect
+    - name: normalize line endings in conanfile
```

Comment:
Why are you doing this only for this one file?
The `.gitattributes` should have done that for the hole repo?

Reply:
Yes, but this is the only file it is necessary for. This file gets hashed by conan to determine the revision number.